### ISSUE TITLE
Sentry Relay for Apple M1 (arm64v8)

### ISSUE THREAD

**Issue by netandreus** [getsentry/self-hosted#1874]
### Problem Statement

Just see in #1538 that there is some ARM support. But the Docker Image https://hub.docker.com/r/getsentry/relay/ does not supported ```arm64v8``` architecture. Where can I find docker image for ```sentry-relay```? Or how I can make it myself?

**Comment by hubertdeng123** [comment]
We do not ship arm images for relay. There is a closed draft PR for shipping the arm image for sentry if that's helpful
https://github.com/getsentry/sentry/pull/38180

**Comment by alekitto** [comment]
For the relay part, I created a modified version of its Dockerfile.
[As explained here](https://github.com/getsentry/relay/issues/1305#issuecomment-1282664435) the current dockerfile is not able to compile an arm version of sentry-relay because is based on CentOS 7 which is too old.

In the same comment, I've published a link to my own dockerhub repository where I publish an updated multi platform version of relay (amd64, arm64 and 386). I personally use it onto an ARM kubernetes cluster, if you want to use it, feel free to do it.
Until an official support for ARM images is missing, I will update that image just a couple of days after the official release.
